config/config.js

```diff
@@ -39,15 +39,8 @@ nconf
   .file({file: path.join(__dirname, 'config.json')})
   // 4. Defaults
   .defaults({
-    // Typically you will create a bucket with the same name as your project ID.
-    CLOUD_BUCKET: '',
```

Comment:
Thing we should just pull the defaults from `config.json` (where they're now sort of documented) instead of having configuration & documentation in multiple places.